pages/services/kubernetes/2.1.1-1.12.5/release-notes/index.md

```diff
@@ -3,19 +3,19 @@ layout: layout.pug
 navigationTitle: Release Notes
 title: Release Notes
 menuWeight: 10
-excerpt: Release notes for DC/OS Kubernetes version 2.1.1-1.12.4
+excerpt: Release notes for DC/OS Kubernetes version 2.1.1-1.12.5
 ---
 
 <!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes-cluster -->
 
-# Version 2.1.1-1.12.4
+# Version 2.1.1-1.12.5
 
 ## Breaking Changes from 1.x
 
-* DC/OS Kubernetes `2.1.1-1.12.4` requires DC/OS 1.12.
-* DC/OS Kubernetes `2.1.1-1.12.4` introduces breaking changes to the way the package works and is deployed.
-  Therefore, it is not possible to upgrade an existing installation of DC/OS Kubernetes to `2.1.1-1.12.4`.
-* Before installing `kubernetes-cluster` package `2.1.1-1.12.4`, the `kubernetes` package must be [installed and running](/services/kubernetes/2.1.1-1.12.4/getting-started/installing-mke/).
+* DC/OS Kubernetes `2.1.1-1.12.5` requires DC/OS 1.12.
+* DC/OS Kubernetes `2.1.1-1.12.5` introduces breaking changes to the way the package works and is deployed.
+  Therefore, it is not possible to upgrade an existing installation of DC/OS Kubernetes to `2.1.1-1.12.5`.
```

Comment:
Same "it is not possible to upgrade from an existing installation of Kubernetes on DC/OS 1.11 to..."